.gitlab-ci.yml

```diff
@@ -0,0 +1,274 @@
+    - time cargo build --release --verbose --workspace
+  after_script:
+    # Prepare artifacts
+    - mkdir -p ./artifacts
+    - strip ./target/release/rialto-bridge-node
+    - mv -v ./target/release/rialto-bridge-node ./artifacts/
+    - strip ./target/release/millau-bridge-node
+    - mv -v ./target/release/millau-bridge-node ./artifacts/
+    - strip ./target/release/ethereum-poa-relay
+    - mv -v ./target/release/ethereum-poa-relay ./artifacts/
+    - strip ./target/release/substrate-relay
+    - mv -v ./target/release/substrate-relay ./artifacts/
+    - mv -v ./deployments/local-scripts/bridge-entrypoint.sh ./artifacts/
+    - mv -v ./ci.Dockerfile ./artifacts/
+
+build-nightly:
+  stage:                           build
+  <<:                              *docker-env
+  <<:                              *collect-artifacts
+  <<:                              *nightly-test
+  script:
+    - rustup default nightly
+    - *build-script
+
+#### stage:                        publish
+
+.build-push-image:                 &build-push-image
+  <<:                              *kubernetes-build
+  image:                           quay.io/buildah/stable
+  <<:                              *build-refs
+  variables:                       &image-variables
+    GIT_STRATEGY:                  none
+    DOCKERFILE:                    ci.Dockerfile
+    IMAGE_NAME:                    docker.io/paritytech/$CI_JOB_NAME
+  needs:
+    - job:                         build
+      artifacts:                   true
+  before_script:                   &check-versions
+    - if [[ "${CI_COMMIT_TAG}" ]]; then
+        VERSION=${CI_COMMIT_TAG};
+      elif [[ "${CI_COMMIT_REF_NAME}" ]]; then
+        VERSION=$(echo ${CI_COMMIT_REF_NAME} | sed -r 's#/+#-#g');
+      fi
+    - echo "Effective tags = ${VERSION} sha-${CI_COMMIT_SHORT_SHA} latest"
+  script:
+    - test "${Docker_Hub_User_Parity}" -a "${Docker_Hub_Pass_Parity}" ||
+        ( echo "no docker credentials provided"; exit 1 )
+    - cd ./artifacts
+    - buildah bud
+        --format=docker
+        --build-arg VCS_REF="${CI_COMMIT_SHORT_SHA}"
+        --build-arg HEALTH="${HEALTH}"
+        --build-arg BUILD_DATE="$(date +%d-%m-%Y)"
+        --build-arg PROJECT="${CI_JOB_NAME}"
+        --build-arg VERSION="${VERSION}"
+        --tag "${IMAGE_NAME}:${VERSION}"
+        --tag "${IMAGE_NAME}:sha-${CI_COMMIT_SHORT_SHA}"
+        --tag "${IMAGE_NAME}:latest"
+        --file "${DOCKERFILE}" .
+    # The job will success only on the protected branch
+    - echo "$Docker_Hub_Pass_Parity" |
+        buildah login --username "$Docker_Hub_User_Parity" --password-stdin docker.io
+    - buildah info
+    - buildah push --format=v2s2 "${IMAGE_NAME}:${VERSION}"
+    - buildah push --format=v2s2 "${IMAGE_NAME}:sha-${CI_COMMIT_SHORT_SHA}"
+    - buildah push --format=v2s2 "${IMAGE_NAME}:latest"
+  after_script:
+    - env REGISTRY_AUTH_FILE= buildah logout "$IMAGE_NAME"
+
+rialto-bridge-node:
```

Comment:
Don't we need to update `docker-compose.yml`?

Reply:
I need to know more about it

Reply:
I thought that the docker image names might have changed and the docker-compose file is fetching images from dockerhub, but I guess there were no changes.